### PR TITLE
Support OGG using SoundManager2

### PR DESCRIPTION
**Pull request by pellaeon**
A new wrapper class `PlayerWrapper` is created, which is used to call underlying Aurora.js or SoundManager2 depending on format.

### CODE REVIEW

File: js/public/app.js [hunk 1]
```diff
@@ -594,7 +594,9 @@ angular.module('Music').factory('ArtistFactory', ['Restangular', '$rootScope', f
 
 angular.module('Music').factory('Audio', ['$rootScope', function ($rootScope) {
 	$rootScope.$emit('SoundManagerReady');
-	return AV.Player;
+	//return AV.Player;
+	return new PlayerWrapper();
+	//return PlayerWrapper;
```
Comment (MorrisJobke) [inline js/public/app.js]: Can you drop these comments here?

File: js/public/app.js [hunk 2]
```diff
@@ -341,8 +341,12 @@ angular.module('Music').controller('PlayerController',
 
 	$scope.getPlayableFileURL = function (track) {
 		for(var mimeType in track.files) {
-			if(mimeType=='audio/flac' || mimeType=='audio/mpeg') {
-				return track.files[mimeType];
+		console.log(mimeType);
```
Comment (MorrisJobke) [inline js/public/app.js]: Can you drop the console.log?

**Comment by MorrisJobke** [comment]
Beside my comments this looks really awesome :) Thanks for the contribution :)

**Comment by pellaeon** [comment]
@MorrisJobke Do you want to merge this PR first or you want to wait after #419 ?

**Comment by pellaeon** [comment]
yay :fireworks: :fireworks:

**Pull request merged**